app/grids/ats/candidates_grid.rb

```diff
@@ -88,19 +88,28 @@ class ATS::CandidatesGrid
     :status,
     :enum,
     select: lambda {
-      Placement.statuses.map { |k, v| [k.humanize, v] }
-                        .insert(1, %w[Disqualified disqualified])
+      mandatory_statuses_for_select =
+        [%w[Qualified qualified], %w[Disqualified disqualified], %w[Reserved reserved]]
+      mandatory_statuses_for_select +
+      DisqualifyReason.pluck(:title).map { [_1, _1.parameterize.underscore] }
```

Comment:
Why do we need parameterize here and humanize inside where?
I assume it won't work is someone call disqualify reason "NEVER  CONTACT!!!!!", or any other not humanized text

Reply:
Won't it be confusing if 3 mandatory statuses will be paired like `[humanized, snake cased]` and others - [humanized, humanized]?

Reply:
Who will be confused?
Actualy, you may pass the reason id to the backend